### ISSUE TITLE
Bounding Box support?

### ISSUE THREAD

**Issue by cody-elhard** [rgeo/rgeo-geojson#58]
When I try to encode the following well known text, I get a nil response.

`RGeo::GeoJSON.encode("BOX(-100 30,-120 30)")`

Should I just convert this to a polygon or should bounding boxes be supported?

**Comment by BuonOmo** [comment]
Yeah `RGeo::GeoJSON.encode` only takes features. You could have some kind of helper to pass through anything, like https://github.com/ekylibre/charta. Although I think it is now unmaintained, you can definitely take inspiration on the `new_feature` method to create a convenient helper in your project (https://github.com/ekylibre/charta/blob/master/lib/charta.rb#L29)

About your mention of bounding boxes, I guess you refer to https://www.rfc-editor.org/rfc/rfc7946#section-5. There is not plan yet to support that, I don't really know how the interface could do that clearly, and I think I'd rather stick to something simple. Although if you have some design proposition in mind, please share it!

**Comment by BuonOmo** [comment]
If anyone is up to convincing that we should do the feature, why not. In the meantime I won't have any time to dedicate to this feature unfortunately
